RELEASING.md

```diff
@@ -15,11 +15,24 @@ Only repo administrator is allowed to run the release script, otherwise it will
 Prior to making a release,
 
 1. Sync any changes that have been merged to backup-utils-private into this repository.
-1. Go through the list of open pull requests and merge any that are ready for merging.
-1. Go through the list of closed pull requests since the last release and ensure those that should be included in the release notes:
-  - have a "bug", "enhancement" or "feature" label,
-  - have a title that clearly describes the changes in that pull request. Reword if necessary.
-1. Perform a dry run (add `--dry-run` to one of the commands below) and verify the version strings are going to be changed and verify the release notes.
+
```

Comment:
I'm not sure if we want these changes in the public repo.

Reply:
Should this file be removed from the public repo?

Reply:
I would vote yes, because we don't make releases from public; we sync it and private and then push the changes out from there.

Reply:
I went ahead and removed it for now.